test.go

```diff
@@ -575,6 +577,11 @@ func (m *mockMsgStatus) ChannelUUID() ChannelUUID { return m.channel.UUID() }
 func (m *mockMsgStatus) ID() MsgID                { return m.id }
 func (m *mockMsgStatus) EventID() int64           { return int64(m.id) }
 
+func (m *mockMsgStatus) OldURN() urns.URN       { return m.oldURN }
+func (m *mockMsgStatus) SetOldURN(urn urns.URN) { m.oldURN = urn }
+func (m *mockMsgStatus) NewURN() urns.URN       { return m.newURN }
+func (m *mockMsgStatus) SetNewURN(urn urns.URN) { m.newURN = urn }
```

Comment:
Since old and new really go together it probably makes sense for these to be more joined.

IE:
  `SetUpdatedURN(old urns.URN, new urns.URN) error`
  `UpdatedURN() (urns.URN, urns.URN)`
  `HasUpdatedURN() bool`